### PR TITLE
Updated Working with Terrain Data for Qgis 3

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: source/docs/3/working_with_terrain.rst [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Working with Terrain Data (QGIS3)
+=================================
+
+Terrain or elevation data is useful for many GIS analyses, and it is often used
+in maps. QGIS 3 has good terrain processing capabilities built-in. In this
+tutorial, we will work through the steps to generate various products from
+elevation data such as hillshade, contours, and contour smoothening, etc.
```
Comment (spatialthoughts) [inline source/docs/3/working_with_terrain.rst:7]: hillshade and contours.

File: source/docs/3/working_with_terrain.rst [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+.. image:: /static/3/working_with_terrain/images/13.png
+   :align: center
+
+14. A new layer ``mt_everest_hillshade`` will appear in the canvas.
+
+.. image:: /static/3/working_with_terrain/images/14.png
+   :align: center
+
+15. Search for Contour in :guilabel:`Processing Toolbox`. Select the ``Contour`` under GDAL algorithms.
```
Comment (spatialthoughts) [inline source/docs/3/working_with_terrain.rst:134]: the contour algorithm.

File: source/docs/3/working_with_terrain.rst (hunk 1 above)
Comment (Santhosh-M31) [reply source/docs/3/working_with_terrain.rst]: Updated

File: source/docs/3/working_with_terrain.rst (hunk 2 above)
Comment (Santhosh-M31) [reply source/docs/3/working_with_terrain.rst]: Updated

**Pull request merged**